src/core/sys/windows/ddeml.d

```diff
@@ -192,11 +207,6 @@ enum : int {
     MH_CLEANUP = 4
 }
 
-mixin DECLARE_HANDLE!("HCONVLIST");
-mixin DECLARE_HANDLE!("HCONV");
-mixin DECLARE_HANDLE!("HSZ");
-mixin DECLARE_HANDLE!("HDDEDATA");
-
```

Comment:
any reason to move this?

Reply:
Yes, `HDDEDATA` is used on row 151.

Reply:
Since it needed to go higher up anyway, I placed it at the same spot as in the Windows header.